src/pages/MyProfilePage.js

```diff
@@ -1,32 +1,68 @@
-import React from "react";
-import "../styles.css";
-import { useSelector } from "react-redux";
+import React from 'react';
+import '../styles.css';
+import { useSelector } from 'react-redux';
 
-const getReseved = (rocket) => {
+const getReserved = (rocket) => {
   if (rocket.reserved) {
     return (
       <tr key={rocket}>
         <td className="separator">{rocket.rocket_name}</td>
       </tr>
     );
   }
+
+  return null;
+  // return (
+
+  // );
```

Comment:
- [ ] [OPTIONAL] Kindly do not leave commented codes in your codebase since it will make it bulky and slow as the project becomes bigger.